docs/Doxyfile

```diff
@@ -58,7 +58,7 @@ PROJECT_LOGO           =
 # entered, it will be relative to the location where doxygen was started. If
 # left blank the current directory will be used.
 
-OUTPUT_DIRECTORY       = ../../pfnet-docs
+OUTPUT_DIRECTORY       = $(PFNET_DOCS)
```

Comment:
This sets the output but the INPUT was not changed from ../ to ./ because now the doxygen is called from a different directory. I am fixing that.